jerry-core/ecma/builtin-objects/ecma-builtin-date.c

```diff
@@ -707,17 +750,19 @@ ecma_builtin_date_dispatch_call (const ecma_value_t *arguments_list_p, /**< argu
 {
   JERRY_UNUSED (arguments_list_p);
   JERRY_UNUSED (arguments_list_len);
-  ecma_value_t ret_value = ECMA_VALUE_EMPTY;
 
-  ECMA_TRY_CATCH (now_val,
-                  ecma_builtin_date_now (ECMA_VALUE_UNDEFINED),
-                  ret_value);
+  ecma_value_t now_val = ecma_builtin_date_now (ECMA_VALUE_UNDEFINED);
```

Comment:
How about we update this function? The argument is marked as unused, so it could be removed, and the function could return with a number instead of an ecma value.